server/models/Address.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require("mongoose");
+
```

Comment:
Could we work on the formatting of this file a bit better and others as well?

Example

```js
const addressSchema = new mongoose.Schema({
    streetNumber: {
        type: Number, 
        required: true
    },
    . . . 
});
```